frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx

```diff
@@ -50,11 +50,10 @@ const PidPinForm: FunctionComponent<PidPinProps> = (props: PidPinProps) => {
   return (
     <>
       <Form.Row className="d-inline-flex flex-nowrap">
-        <Form.Label className="required">PID</Form.Label>
+        <Form.Label className="req">*</Form.Label>PID
```

Comment:
Seems odd to have the label outside like this?

Reply:
Agreed, but this is how it is in current dev-alpha